sklearn/ensemble/_forest.py

```diff
@@ -1004,7 +1005,10 @@ class RandomForestClassifier(ForestClassifier):
     n_jobs : int, default=None
         The number of jobs to run in parallel. :meth:`fit`, :meth:`predict`,
         :meth:`decision_path` and :meth:`apply` are all parallelized over the
-        trees. ``None`` means 1 unless in a :obj:`joblib.parallel_backend`
+        trees. This works by creating n_jobs sets of trees and computing them
+        in parallel.
```

Comment:
that's a little bit ambiguous IMHO, also I'm not sure that's what happens in practice. Did you want to refer to the type of scheduling that is applied, e.g. dynamic vs static scheduling?

Reply:
Thanks, @NicolasHug ! Can definitely clarify.

I hadn't thought to refer to static v dynamic scheduling -- we looked at the some of the documentation that was marked "completed" in #14228, and it didn't seem to include that level of detail. Do you have a recommendation for where to look at an example ?

Reply:
unfortunately the list in https://github.com/scikit-learn/scikit-learn/issues/14228#issuecomment-534841015 doesn't seem up to date 

and in fact the docstring for the random forest was already updated in https://github.com/scikit-learn/scikit-learn/pull/14628

If you think the current version isn't clear enough we can definitly improve it though. What did you find confusing / unclear?

Reply:
Thanks, @NicolasHug ! I also thought the documentation was mostly quite clear -- our addition was just meant to add a bit more detail about how tree models themselves are parallelized, since the current description focused on the class methods. 

But that might be the more sensible description ! Happy to close this if you think it's confusing.

Reply:
Got it, yeah in this case I think this could be closed. Sorry for the extra work!